scripts/plugins.txt

```diff
@@ -0,0 +1,22 @@
+ws-cleanup:0.36
+timestamper:1.8.10
+thinBackup:1.9
+ssh-agent:1.16
+saferestart:0.3
+role-strategy:2.9.0
+openstack-cloud:2.40
+nodejs:1.2.6
+kerberos-sso:1.4 
+junit-attachments:1.5
+http_request:1.8.22
+groovy:2.0 
+gitlab-plugin:1.5.9
+gitlab-hook:1.4.2
+ghprb:1.42.0
+github-branch-source:2.3.6
+extended-choice-parameter:0.76
+delivery-pipeline-plugin:1.2.0
+cvs:2.14
+aws-sqs:2.0.1
+ansicolor:0.5.2
+ansible-tower:0.9.1
```

Comment:
@mikenairn @trepel Are all the required plugins included in this file? Are there any here that are not needed and can be removed?

Reply:
@JameelB Currently installed plugins are https://gist.github.com/mikenairn/acce9a8150c3d864fce046b1b150a0fd

You won't need to list all of them in here as a lot will be installed as part of Jenkins, but it might give you an idea of some that we rely on that might be missing from this list

Reply:
I don't really know what plugins are required for the tests, sorry. I can remember us installing a few, but nothing crazy. I think the best way to proceed is to just proceed with what is here now and if it turns out the test suites require some other plugin, we will update the list.

Reply:
Did a quick check and these seem to be the plugins we have installed that aren't installed as part of jenkins:

```
< ansible-tower
< ant
< aws-sqs
< build-pipeline-plugin
< cloud-stats
< command-launcher
< cvs
< delivery-pipeline-plugin
< email-ext
< embeddable-build-status
< external-monitor-job
< ghprb
< gitlab-hook
< gitlab-plugin
< groovy
< jms-messaging
< junit-attachments
< kerberos-sso
< ldap
< nodejs
< openstack-cloud
< pam-auth
< redhat-ci-plugin
< role-strategy
< ruby-runtime
< saferestart
< thinBackup
< timestamper
< windows-slaves
```

I think you have most, if not all covered, we can obviously add any as we find them.